tests/Unit/Domain/Regulation/MeasureTest.php

```diff
@@ -35,8 +36,8 @@ public function testMeasure(): void
         $this->assertSame($date, $measure->getCreatedAt());
```

Comment:
On peut vérifier que la maxSpeed est bien null

```suggestion
        $this->assertNull($measure->getMaxSpeed());
        $this->assertSame($date, $measure->getCreatedAt());
```